taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = (
+            "username",
+            # "password",
+            "first_name",
+            "last_name",
+            "license_number",
+            "email"
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = (
+            "license_number",
+        )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if (len(license_number) != 8
+                or not license_number[:3].isupper()
+                or not license_number[:3].isalpha()
+                or not license_number[3:].isdigit()):
+            raise ValidationError("Ensure that value: "
+                                  "сonsist only of 8 characters, "
+                                  "first 3 characters are uppercase letters, "
+                                  "last 5 characters are digits")
+
+        return license_number
```

Comment:
validation should be also on creation, if I'm not wrong !?

Reply:
U r right. Fixed